src/test/java/ebl/v2/ShippingInstructionIT.java

```diff
@@ -98,4 +104,61 @@ void testFailureOnNonExistingCarrBookingRefPostShippingInstruction() {
         .body()
         .asString();
   }
+
+  @Test
+  void testInvalidGetShippingInstruction() {
+
+    UUID invalidShippingInstructionReference = UUID.randomUUID();
+
+    given()
+        .contentType("application/json")
+        .get(SHIPPING_INSTRUCTIONS + "/" + invalidShippingInstructionReference)
+        .then()
+        .assertThat()
+        .statusCode(HttpStatus.SC_NOT_FOUND)
+        .body("httpMethod", equalTo("GET"))
+        .body("requestUri", containsString("/v2/shipping-instructions"))
```

Comment:
I don't think a valid response returns lines 119 & 120.

Reply:
It was supposed to be part of an invalid usecase, but I created the PR way too early without setting it to draft first...